runtime/vm/jfr.cpp

```diff
@@ -1152,6 +1162,145 @@ jfrDump(J9VMThread *currentThread, BOOLEAN finalWrite)
 	flushAllThreadBuffers(currentThread, finalWrite);
 	writeOutGlobalBuffer(currentThread, finalWrite);
 }
+
+UDATA
+jfrTypeIDHashFn(void *key, void *userData)
+{
+	const J9JFRTypeID *entry = (J9JFRTypeID *) key;
```

Comment:
There's no need to involve non-const types:
```c
	const J9JFRTypeID *entry = (const J9JFRTypeID *)key;
```
Same comment applies to lines 1179-1180.